src/main/java/com/jfrog/ide/common/components/DependencyNode.java

```diff
@@ -0,0 +1,132 @@
+package com.jfrog.ide.common.components;
+
+import com.jfrog.ide.common.components.subentities.License;
+import com.jfrog.ide.common.components.subentities.Severity;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.swing.tree.TreeNode;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.jfrog.ide.common.utils.Utils.removeComponentIdPrefix;
+
+/**
+ * @author yahavi
+ */
+public class DependencyNode extends ComparableSeverityTreeNode implements Serializable, SubtitledTreeNode {
+    private static final long serialVersionUID = 1L;
```

Comment:
I think we can take the chance and get rid of the Serializable. We use Jackson and not serialization.
```suggestion
public class DependencyNode extends ComparableSeverityTreeNode implements SubtitledTreeNode {
```

Reply:
It's going to be used again really soon, when the cache will be back.